### DISCUSSION TITLE
完全没法用啊

### DISCUSSION THREAD

**Discussion by Bolganoth** [DumzGW/-WayfinderSaveEditor-python-#1] [General]
选了存档文件，点了高级修改，没有一点反应。
多点几次高级修改再点Uncompress还会直接关掉。
有在空环境的windows上测试过运行吗？

**Comment by isoTOP-e** [comment]
How is this save editor supposed to work? I run SaveEditor.exe, select a save to unpack, but nothing happens. What's the problem?
